components/InputManual.js

```diff
@@ -1 +1,41 @@
-// sdgsd
+import React, { Component } from 'react';
+import { Text, TextInput, View, StyleSheet, Alert, TouchableOpacity, Button } from 'react-native';
+import { createStackNavigator, StackActions, NavigationActions } from 'react-navigation';
+
+class InputManual extends React.Component {
+  state = {
+    barCodeNumber: ""
+  };
+
+  handleBarCode = (text) => {
+    this.setState({barCodeNumber: text})
```

Comment:
That's great - then it should integrate with Results.js using url = InputManual.state.barCodeNumber 👍

Reply:
Consistency with barcode number stored state?